### PR TITLE
Add popup max-width option

### PR DESCRIPTION
**Pull request by arinaturcu**
Adds a Range Control for setting the max-width property on the Popup block.
It's located under the min-width control in the `Style` pannel.

### CODE REVIEW

File: src/blocks/blocks/popup/block.json [hunk 1]
```diff
@@ -12,6 +12,10 @@
 			"type": "string"
 		},
 		"minWidth": {
+			"type": "number",
+			"default": 400
```
Comment (HardeepAsrani) [inline src/blocks/blocks/popup/block.json:16]: @arinaturcu Might be better to add the default via CSS and leave this empty.

File: src/blocks/blocks/popup/inspector.js [hunk 2]
```diff
@@ -20,6 +20,13 @@ import {
 
 import { Fragment } from '@wordpress/element';
 
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+
+const { attributes: defaultAttributes } = metadata;
+
```
Comment (HardeepAsrani) [inline src/blocks/blocks/popup/inspector.js:29]: Any reason to import metadata here? Don't see it being anywhere..

Comment (arinaturcu) [reply src/blocks/blocks/popup/inspector.js]: No, just forgot it there.

**Comment by HardeepAsrani** [comment]
Seems all good from my side. I've put it on QA for Irinel to make sure I haven't missed anything. Thank you!

**Comment by irinelenache** [comment]
@arinaturcu Tested the feature and didn't find any problem 🚀

**Pull request merged**